src/main/java/fi/oph/henkilotietomuutospalvelu/service/parse/TietoryhmaSerializerUtil.java

```diff
@@ -0,0 +1,63 @@
+package fi.oph.henkilotietomuutospalvelu.service.parse;
+
+import fi.oph.henkilotietomuutospalvelu.model.tietoryhma.*;
+
+public class TietoryhmaSerializerUtil {
+
+    public static String serializeTietoryhma(Tietoryhma tietoryhma) {
```

Comment:
Could be cleaner....
1. define "Parser" interface with two methods `Tietoryhma parse(String value)` and `String serialize(Tietoryhma value)`
2. attach parsers as instance variables to ryhmatunnus in enum definition
3. use as: `tietoryhma.getRyhmatunnus().serialize(tietoryhma);`